mp/src/game/shared/da/da_player_shared.cpp

```diff
@@ -799,7 +799,7 @@ void CDAPlayerShared::StartSliding(bool bDiveSliding)
 void CDAPlayerShared::EndSlide()
 {
 	// If it was long enough to notice what it was, then train the slide.
-	if (gpGlobals->curtime - m_flSlideTime > 1)
+	if (m_pOuter->GetCurrentTime() - m_flSlideTime > 1)
```

Comment:
Ideally this should be in real time, not in slow time, because we're measuring whether the player (who lives in real time) had enough time to respond. But since the rest of the code is in slow time this is the correct code. Anyway, this just means that the conditional will trigger strictly after it needs to, which is OK for the spirit of the thing.